packages/core/src/interactions/packet/index.spec.ts

```diff
@@ -25,6 +25,7 @@ import assert from 'assert'
 import { AcknowledgementChallenge, Packet, Acknowledgement } from '../../messages'
 import { PacketForwardInteraction } from './forward'
 import { initializeCommitment } from '@hoprnet/hopr-core-ethereum'
+import { ChannelCommitmentInfo } from '@hoprnet/hopr-core-ethereum/lib/commitment'
```

Comment:
importing from `package/directory` is dangerous

Reply:
```suggestion
import { ChannelCommitmentInfo } from '@hoprnet/hopr-core-ethereum'
```

Reply:
Thanks, missed that one!